src/wmtk/TetMeshOperationExecutor.cpp

```diff
@@ -213,7 +213,7 @@ TetMesh::TetMeshOperationExecutor::get_split_simplices_to_delete(
     return ids;
 }
 
-const std::array<std::vector<long>, 4>
+const std::pair<std::array<std::vector<long>, 4>, std::array<std::vector<Tuple>, 4>>
```

Comment:
returning all this is an optimization that isn't clear / obfuscates the code a bit imo